code/game/machinery/autolathe_datums.dm

```diff
@@ -456,3 +456,55 @@
 	name = "miniature cell"
 	path = /obj/item/weapon/cell/device/emergency_light/empty
 	category = "Engineering"
+	
+/datum/autolathe/recipe/cable_coil
+	name = "cable coil"
+	path = /obj/item/stack/cable_coil
+	category = "Devices and Components"
+	is_stack = 1
+
+/datum/autolathe/recipe/labeler
+	name = "hand labeler"
+	path = /obj/item/weapon/hand_labeler
+	category = "General"
+
+/datum/autolathe/recipe/destTagger
+	name = "destination tagger"
+	path = /obj/item/device/destTagger
+	category = "General"
+
+/datum/autolathe/recipe/implanter
```

Comment:
Make this a illegal item for the protolathe

Reply:
Ah yeah the implanter?